docker/android.cmake

```diff
@@ -0,0 +1,33 @@
+# toolchain file for android targets, see #1110
+
+set(CMAKE_SYSTEM_NAME Android)
+set(CMAKE_SYSTEM_PROCESSOR "$ENV{CROSS_ANDROID_ARCH}")
+set(CMAKE_ANDROID_STANDALONE_TOOLCHAIN /android-ndk)
```

Comment:
Since we use a standalone Android NDK with the `/android-ndk/sysroot` with no version prefix, we need to use `CMAKE_ANDROID_STANDALONE_TOOLCHAIN ` and not `CMAKE_SYSROOT` nor `CMAKE_ANDROID_NDK`.

See the docs [here](https://cmake.org/cmake/help/latest/manual/cmake-toolchains.7.html#cross-compiling-for-android).